pkg/querier/querier.go

```diff
@@ -683,8 +680,8 @@ func partitionChunks(chunks []chunk.Chunk, mint, maxt int64, iteratorFunc chunkI
 	for i := range chunksBySeries {
 		series = append(series, &storage.SeriesEntry{
 			Lset: chunksBySeries[i][0].Metric,
-			SampleIteratorFn: func(_ chunkenc.Iterator) chunkenc.Iterator {
-				return iteratorFunc(chunksBySeries[i], model.Time(mint), model.Time(maxt))
+			SampleIteratorFn: func(it chunkenc.Iterator) chunkenc.Iterator {
+				return iteratorFunc(it, chunksBySeries[i], model.Time(mint), model.Time(maxt))
```

Comment:
Not blocker. It would be best to get rid of this function entirely as it is used in unit tests only. Still legacy code